templates/icinga.ini

```diff
@@ -300,3 +300,277 @@ yUnitSystem = "none"
 [icinga-graphite-queue.functions]
 graphitewriter_graphite_work_queue_items.value = "alias(color($metric$, '#1a7dd7'), 'Items')"
 graphitewriter_graphite_work_queue_item_rate.value = "alias(color($metric$, '#0b3c68'), 'Items/s')"
+
+
+[icinga-active-host-checks.graph]
+check_command = "icinga"
+
+[icinga-active-host-checks.metrics_filters]
+graphitewriter_active_host_checks_1min.value = "$service_name_template$.perfdata.active_host_checks_1min.value"
+graphitewriter_active_host_checks_5min.value = "$service_name_template$.perfdata.active_host_checks_5min.value"
+graphitewriter_active_host_checks_15min.value = "$service_name_template$.perfdata.active_host_checks_15min.value"
+
+[icinga-active-host-checks.urlparams]
+title = "Active Host Checks"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-active-host-checks.functions]
+graphitewriter_active_host_checks_1min.value = "alias(color($metric$, '#61aaed'), '1min')"
+graphitewriter_active_host_checks_5min.value = "alias(color($metric$, '#1a7dd7'), '5min')"
+graphitewriter_active_host_checks_15min.value = "alias(color($metric$, '#0b3c68'), '15min')"
+
+
+[icinga-active-service-checks.graph]
+check_command = "icinga"
+
+[icinga-active-service-checks.metrics_filters]
+graphitewriter_active_service_checks_1min.value = "$service_name_template$.perfdata.active_service_checks_1min.value"
+graphitewriter_active_service_checks_5min.value = "$service_name_template$.perfdata.active_service_checks_5min.value"
+graphitewriter_active_service_checks_15min.value = "$service_name_template$.perfdata.active_service_checks_15min.value"
+
+[icinga-active-service-checks.urlparams]
+title = "Active Service Checks"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+    
+[icinga-active-service-checks.functions]
+graphitewriter_active_service_checks_1min.value = "alias(color($metric$, '#61aaed'), '1min')"
+graphitewriter_active_service_checks_5min.value = "alias(color($metric$, '#1a7dd7'), '5min')"
+graphitewriter_active_service_checks_15min.value = "alias(color($metric$, '#0b3c68'), '15min')"
+
+
+[icinga-api-num-json-rpc-relay-queue.graph]
+check_command = "icinga"
+
+[icinga-api-num-json-rpc-relay-queue.metrics_filters]
+graphitewriter_api_num_json_rpc_relay_queue_items.value = "$service_name_template$.perfdata.api_num_json_rpc_relay_queue_items.value"
+graphitewriter_api_num_json_rpc_relay_queue_item_rate.value = "$service_name_template$.perfdata.api_num_json_rpc_relay_queue_item_rate.value"
+
+[icinga-api-num-json-rpc-relay-queue.urlparams]
+title = "API Num JSON RPC Relay Queue"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-api-num-json-rpc-relay-queue.functions]
+graphitewriter_api_num_json_rpc_relay_queue_items.value = "alias(color($metric$, '#61aaed'), 'Items')"
+graphitewriter_api_num_json_rpc_relay_queue_item_rate.value = "alias(color($metric$, '#0b3c68'), 'Item Rate')"
```

Comment:
1. IMO "Num" isn't needed, but a better unit in the legend.
2. Why no *Count* metric?